tests/network/flat_overlay/test_flat_overlay.py

```diff
@@ -8,10 +8,13 @@
 LOGGER = logging.getLogger(__name__)
 
 
-pytestmark = pytest.mark.usefixtures(
-    "skip_if_no_multinic_nodes",
-    "enable_multi_network_policy_usage",
-)
+pytestmark = [
+    pytest.mark.special_infra,
```

Comment:
I don't think that the setup for this module should be considered as special-infra.

Reply:
@yossisegev , I think it was done to cover jumbo frames setups.
Do all setups support jambo frames?
Do you have a different suggestion?

Reply:
@yossisegev, @EdDev is correct. If you need jumbo frame, it requires special config.

Reply:
I think that most of our clusters - including PSI - support jumbo. AFAIK DevOps don't apply any special configuration in order to deploy a jumbo-supporting cluster.

Reply:
Can we ref someone that can confirm this for us?
The support for Jumbo frames requires the inter-node connectivity to support it as well.

Reply:
If you check deployment jobs for both BM and PSI, you would see `JUMBO_FRAME` option only exists for BM clusters. It requires special configuration.

Reply:
As we discussed in slack - jumbo support requires a dedicated deployment option only for BM.
For PSI - we get it by default.